src/core/ext/transport/chttp2/transport/message_assembler.h

```diff
@@ -61,7 +61,9 @@ class GrpcMessageAssembler {
       return ReturnNullOrError();
     }
     GrpcMessageHeader header = ExtractGrpcHeader(message_buffer_);
-    if (message_buffer_.Length() >= header.length + kGrpcHeaderSizeInBytes) {
```

Comment:
Another way to do this would be to say `message_buffer_.Length() - kGrpcHeaderSIzeInBytes >= header.length`